src/main/java/com/wavefront/sdk/jaxrs/server/WavefrontJaxrsServerFilter.java

```diff
@@ -205,14 +207,18 @@ private void processResponse(ContainerRequestContext containerRequestContext,
                                ContainerResponseContext containerResponseContext) {
     if (tracer != null) {
       try {
-        Scope scope = (Scope) containerRequestContext.getProperty(PROPERTY_NAME);
+        SpanWrapper spanWrapper = (SpanWrapper) containerRequestContext.getProperty(PROPERTY_NAME);
+        Scope scope = spanWrapper.getScope();
```

Comment:
you need to check if spanWrapper is null

Reply:
fixed, thanks!